www/js/localstorage.js

```diff
@@ -0,0 +1,36 @@
+'user strick';
+
+angular.module('emission.main.localstorage',['angularLocalStorage'])
```

Comment:
nice! I also have wrappers for the local storage, but I create modules based on where they are used. With that pattern, you would use a CalorieCalc module instead, which would encapsulate load, store and calculation.